src/main/java/io/split/android/client/utils/Utils.java

```diff
@@ -84,4 +84,20 @@ public static boolean isReachable(String host, int port, int timeoutMs ) {
             return true;
         } catch (IOException e) { return false; }
     }
+
+    public static String sanitizeForFileName(String string) {
+        if(string == null) {
+            return "";
+        }
+        return string.replaceAll("[^a-zA-Z0-9\\.\\-]", "_");
+    }
+
+    public static String join(String separator, Iterable<String> iterable) {
+        StringBuilder builder = new StringBuilder();
+        for(String string : iterable) {
+            builder.append(string).append(",");
```

Comment:
what happens with separator?

Reply:
Is not a different way with Strings.join (or Stirng.join) so you dont need this?

Reply:
Function was unused and removed!